apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/faucet/components/SpinWheel.tsx

```diff
@@ -0,0 +1,176 @@
+"use client";
+
+import { CopyTextButton } from "@/components/ui/CopyTextButton";
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+import { THIRDWEB_ENGINE_FAUCET_WALLET } from "@/constants/env";
+import { CustomConnectWallet } from "@3rdweb-sdk/react/components/connect-wallet";
+import type { ChainMetadataWithServices } from "app/(dashboard)/(chain)/types/chain";
+import { thirdwebClient } from "lib/thirdweb-client";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import Confetti from "react-confetti";
+import { toast } from "sonner";
+import { type ISpinWheelProps, SpinWheel } from "spin-wheel-game";
+import { defineChain } from "thirdweb";
+import { useActiveAccount, useWalletBalance } from "thirdweb/react";
+import { Text } from "tw-components";
```

Comment:
Avoid using chakra components and use tailwind instead

Reply:
removed